src/routes/device-logs.ts

```diff
@@ -216,6 +219,13 @@ export const store: RequestHandler = async (req: Request, res: Response) => {
 		const logs: DeviceLog[] = supervisor.convertLogs(ctx, body);
 		if (logs.length) {
 			await getBackend(ctx).publish(ctx, logs);
+			try {
+				if (LOKI_HOST) {
+					await loki.publish(ctx, logs);
+				}
+			} catch (err) {
+				captureException(err, 'Failed to publish log to Loki');
+			}
```

Comment:
I think it makes sense to put the entire try/catch inside the if, ie
```suggestion
			if (LOKI_HOST) {
				try {
					await loki.publish(ctx, logs);
				} catch (err) {
					captureException(err, 'Failed to publish log to Loki');
				}
			}
```

Reply:
updated